datatype.go

```diff
@@ -77,3 +78,68 @@ func dataTypeMode(v driver.Value) (tsmode string, err error) {
 	}
 	return tsmode, nil
 }
+
+// SnowflakeParameter includes the columns output from SHOW PARAMETER command.
+type SnowflakeParameter struct {
+	Key              string
+	Value            string
+	Default          string
+	Level            string
+	Description      string
+	SetByUser        string
+	SetInJob         string
+	SetOn            string
+	SetByThreadID    string
+	SetByThreadName  string
+	SetByClass       string
+	ParameterComment string
+}
+
+func populateSnowflakeParameter(colname string, p *SnowflakeParameter) interface{} {
+	switch colname {
+	case "key":
+		return &p.Key
+	case "value":
+		return &p.Value
+	case "default":
+		return &p.Default
+	case "level":
+		return &p.Level
+	case "description":
+		return &p.Description
+	case "set_by_user":
+		return &p.SetByUser
+	case "set_in_job":
+		return &p.SetInJob
+	case "set_on":
+		return &p.SetOn
+	case "set_by_thread_id":
+		return &p.SetByThreadID
+	case "set_by_thread_name":
+		return &p.SetByThreadName
+	case "set_by_class":
+		return &p.SetByClass
+	case "parameter_comment":
+		return &p.ParameterComment
+	default:
+		panic("unknown type " + colname)
+	}
+}
```

Comment:
This is required to support both the standard and extended SHOW PARAMETER output.